src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -125,6 +125,8 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			generateScriptEvent.withAdditionalMeasurements({
 				'elapsedTimeMs': new Date().getTime() - startTime
 			}).send();
+			// Close notification in 1 second.
```

Comment:
It's generally good to explain why (especially since the code itself is pretty clear about what it's doing in this case)

Also - it looks like we aren't doing anything when an error occurs. Can you double check that to make sure we don't have the toast continuing to show in that case as well?